modules/ipfs-cpinner-client/README.md

```diff
@@ -1,3 +1,180 @@
-# IPFS centralized pinner client
+<p align="middle">
+    <img src="https://www.rifos.org/assets/img/logo.svg" alt="logo" height="100" >
+</p>
+<h3 align="middle"><code>@rsksmart/ipfs-cpinner-client</code></h3>
+<p align="middle">
+    RIF Identity - IPFS Centralized Pinner Client
+</p>
+<p align="middle">
+  <a href="https://rsksmart.github.io/rif-identity-docs/data-vault/cpinner/cpinner-client">
+    <img src="https://img.shields.io/badge/-docs-brightgreen" alt="docs" />
+  </a>
+</p>
 
-_This API is still incompatible with current service_
+A Web Client to simplify the way the services provided by the IPFS Centralized Data Vault Service are consumed.
+
+## Features
+
+- Manage authentication according to the [DID Auth protocol](https://rsksmart.github.io/rif-identity-docs/ssi/libraries/express-did-auth)
+
+- CRUD operations over the RIF Data Vault
+
+- Stores the authentication credentials in the given storage 
+
+## Usage
```

Comment:
I would move all this section directly to docs. It is very big for a readme.

```suggestion
## Usage

[Read the docs](https://rsksmart.github.io/rif-identity-docs/data-vault/cpinner/cpinner-client)

```

I will then review and iterate docs over that PR